lib/OpenQA/WebAPI/Plugin/Gru.pm

```diff
@@ -105,7 +105,10 @@ sub enqueue {
         });
 
     $self->app->minion->enqueue(
-        $task => $args => {priority => $options->{priority} // 0, delay => $delay, notes => {gru_id => $gru->id}});
+        $task => $args => {
+            priority => $options->{priority} // 0,
+            delay    => $delay,
+            notes    => {gru_id => $gru->id, (ttl => $ttl) x !!(defined $ttl)}});
```

Comment:
`(ttl => $ttl) x !!(defined $ttl)` is very creative, props for that 😃. But maybe `defined $ttl ? (ttl => $ttl) : ()` would be easier to understand for less advanced Perl programmers?